aion/web/elm/src/Room/Api.elm

```diff
@@ -1,19 +1,20 @@
 module Room.Api exposing (..)
 
-import General.Constants exposing (hostname)
 import Http
 import Msgs exposing (Msg)
+import Navigation exposing (Location)
 import RemoteData
 import Room.Decoders exposing (roomsDecoder)
+import Urls exposing (host)
 
 
-fetchRoomsUrl : String
-fetchRoomsUrl =
-    hostname ++ "api/subjects"
+fetchRoomsUrl : Location -> String
+fetchRoomsUrl location =
+    (host location) ++ "api/subjects"
```

Comment:
This is now `api/rooms`. Be careful with rebase, ping me when do this if you would like me to check if all is correct.

Reply:
Will ping you for another review once I'm done with this.